src/main.cpp

```diff
@@ -1,8 +1,7 @@
 /***************************************************************************
  *   Copyright (C) 2008-2013 by Heiko Koehn - KoehnHeiko@googlemail.com    *
- *   Copyright (C) 2013-2014, 2016 by Stephen Lyons                        *
- *                                            - slysven@virginmedia.com    *
- *   Copyright (C) 2014 by Ahmed Charles - acharles@outlook.com            *
+ *   Copyright (C) 2013-2016 by Stephen Lyons - slysven@virginmedia.com    *
```

Comment:
OK. BTW if you look very closely at my Gravitar see if you can see what it is holding under it's arm/wing... :grin: